client/src/pages/GameCard.js

```diff
@@ -1,25 +1,20 @@
 import React from 'react'
 let style = {
-  height: '10em',
-  overflowY: 'scroll',
```

Comment:
if I'm not mistaken, this is the portion that makes the boxes with the game description able to scroll instead of taking up a ton of room. so might wanna leave that.

Reply:
this style was written but never actually used. so i just renamed "style2" to "style" and removed the stuff that wasn't used.